src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -1,81 +1,343 @@
+                    flexWrap: 'wrap',
+                  }}
+                >
+                  <Item style={{ flex: '3', paddingBottom: '30px' }}>
+                    <BookingCalendar
+                      style={{
+                        width: '300px',
+                        border: '1px solid #f0f0f0',
+                        borderRadius: '2px',
+                      }}
+                    />
+                  </Item>
+
+                  <Item
+                    name={'availability'}
+                    style={{
+                      display: 'flex',
+                      // flexWrap: 'wrap',
+                      minWidth: '450px',
+                      flexDirection: 'column',
+                      flex: '1',
+                    }}
+                  >
+                    <div
+                      style={{
+                        display: 'flex',
+                        // flexWrap: 'wrap',
+                        justifyContent: 'space-around',
+                      }}
+                    >
+                      <button
+                        style={{
+                          borderRadius: '10px',
+                          padding: '10px 2px',
+                          margin: '6px',
+                          flex: '1',
+                          backgroundColor: '#3BC9B5',
+                          fontSize: '16px',
+                          fontWeight: 'bold',
+                          color: '#680049',
+                        }}
+                      >
+                        9:00 AM
+                      </button>
+                      <button
+                        style={{
+                          borderRadius: '10px',
+                          padding: '10px 2px',
+                          margin: '6px',
+                          flex: '1',
+                          backgroundColor: '#3BC9B5',
+                          fontSize: '16px',
+                          fontWeight: 'bold',
+                          color: '#680049',
+                        }}
+                      >
+                        10:00 AM
+                      </button>
+                      <button
+                        style={{
+                          borderRadius: '10px',
+                          padding: '10px 2px',
+                          margin: '6px',
+                          flex: '1',
+                          backgroundColor: '#3BC9B5',
+                          fontSize: '16px',
+                          fontWeight: 'bold',
+                          color: '#680049',
+                        }}
+                      >
+                        11:00 AM
+                      </button>
+                    </div>
+                    <div
+                      style={{
+                        display: 'flex',
+                        flexWrap: 'wrap',
+                      }}
+                    >
+                      <button
+                        style={{
```

Comment:
Let's make these styles 'DRY' and not inline. :)

Reply:
Oh yes we plan export the styles and make them DRY, we were just using everything inline to prevent switching between screens so much while implementing the figma design. We have actually switched to a dropdown on our current branch. We are implementing ant design styles as well! Thank you for your input Ryan!